pom.xml

```diff
@@ -68,37 +52,10 @@
       <version>${rest-assured.version}</version>
       <scope>test</scope>
     </dependency>
-
-    <dependency>
-      <groupId>com.fasterxml.jackson.core</groupId>
-      <artifactId>jackson-core</artifactId>
-      <version>${jackson.version}</version>
-    </dependency>
-    <dependency>
-      <groupId>com.fasterxml.jackson.core</groupId>
-      <artifactId>jackson-databind</artifactId>
-      <version>${jackson.version}</version>
-    </dependency>
```

Comment:
`mvn dependency:tree -Dincludes=com.fasterxml.jackson.core` run on this pull request reports
```
[INFO] org.folio:mod-event-config:jar:1.4.0-SNAPSHOT
[INFO] \- org.folio:domain-models-runtime:jar:28.0.0:compile
[INFO]    +- com.fasterxml.jackson.core:jackson-core:jar:2.9.9:compile
[INFO]    +- com.fasterxml.jackson.core:jackson-databind:jar:2.9.10:compile
[INFO]    \- com.fasterxml.jackson.core:jackson-annotations:jar:2.9.9:compile
```
jackson-databin:2.9.10 has security vulnerabilities: https://issues.folio.org/browse/FOLIO-2341
One option is to update to RMB 29.0.0 that will be available in a few days and uses jackson 2.10.0.
The other option is to keep the jackson-databind entry in this pom.xml and use either 2.9.10.1 or 2.10.0.
Which option do you choose?

Reply:
This change was made based on comment here https://issues.folio.org/browse/MODCAL-54
ackson-databind 2.10.0 is used by RMB since RMB version 27.1.0
i would prefer to use jackson as transitive dependency

Reply:
Ok, so mod-event-config will update to RMB 29.0.0 when it becomes available.